docs/build/active-learning/index.md

```diff
@@ -0,0 +1,125 @@
+---
+icon: fontawesome/solid/wand-magic-sparkles
+subtitle: Use AI/ML to learn linking rules
+status: new
+tags:
+  - BeginnersTutorial
+---
+
+# Active Learning of Linking Rules
+
+## Introduction
+
+Active learning infuses expert knowledge and creates new relationships between properties of two datasets. We can learn new rules and refine existing rules.
+
+The documentation consists of the following parts described in detail below:
+
+-   [Introduction](#introduction)
+-   [Usage](#usage)
+-   [Start the learning dialog](#start-the-learning-dialog)
+-   [Creating an automatic link rule](#creating-an-automatic-link-rule)
+-   [Add property paths for both entities](#add-property-paths-for-both-entities)
+
+## Usage
+
+Define properties to compare between entities of the selected datasets.
+
+## Start the learning dialog
```

Comment:
IMHO this is not enough to give the user a starting point.

Reply:
extended.